p2panda-rs/src/document/document.rs

```diff
@@ -1,12 +1,71 @@
 // SPDX-License-Identifier: AGPL-3.0-or-later
 
-use std::convert::TryFrom;
+use std::collections::BTreeMap;
 
-use crate::document::{DocumentBuilderError, DocumentView};
+use crate::document::DocumentBuilderError;
 use crate::graph::Graph;
 use crate::hash::Hash;
 use crate::identity::Author;
-use crate::operation::{AsOperation, OperationWithMeta};
+use crate::operation::{AsOperation, OperationValue, OperationWithMeta};
+
+use super::document_view::DocumentViewId;
+use super::DocumentView;
```

Comment:
Aren't both coming from the same place? Maybe theu can be imported in the same line.

Or use the crate level import in line 5?